src/Components/Server/test/Circuits/RevalidatingServerAuthenticationStateProvider.cs

```diff
@@ -208,6 +211,7 @@ public async Task IfValidateAuthenticationStateAsyncReturnsUnrelatedCancelledTas
 
             // Assert: Since we didn't ask for that operation to be canceled, this is treated as
             // a failure to validate, so we force a logout
+            await incrementExecuted.Task.TimeoutAfter(TimeSpan.FromSeconds(1));
```

Comment:
You might want to give just a few extra seconds. It'll be frustrating to see test failures just because the CI agent is under heavy load.

Reply:
Agree